lib/container.js

```diff
@@ -181,6 +181,16 @@ export default class Container {
     });
   }
 
+  whoami() {
+    let container = this;
+    return container.makeRequest('me', {})
+    .then(function (res) {
+      return new container.User(res.result);
+    }, function (err) {
+      return Promise.reject(err);
+    });
```

Comment:
Do you expect the `me` request will return the access token ? 

Currently, the `access_token` field is a required field in the view of SDK, but the `me` endpoint will not return it. Should I inject the token back the returned object and set it to `_currentUser` ?